src/method.c

```diff
@@ -33,7 +33,7 @@ jl_value_t *jl_resolve_globals(jl_value_t *expr, jl_module_t *module, jl_svec_t
         }
         else {
             if (e->head == call_sym && jl_expr_nargs(e) == 3 &&
-                    jl_is_quotenode(jl_exprarg(e, 2)) && module != NULL) {
+                jl_is_quotenode(jl_exprarg(e, 2)) && module != NULL) {
```

Comment:
I think it's clearer when line continuations of block openers are indented an extra level, otherwise the condition and the block are harder to tell apart